src/client/swagger/unity.yml

```diff
@@ -1162,55 +1168,26 @@ components:
         id:
           description: the idpe id of the user
           type: string
+        # TODO(ariel): find out if these are account or user specific
         firstName:
           type: string
+        # TODO(ariel): find out if these are account or user specific
         lastName:
           type: string
         email:
+          # TODO(ariel): find out about this one
+          description: Not sure if this is the billing account email or the account email, but this might be unnecessary since that should be included in the Account or nested in the Account
```

Comment:
not sure what the `firstName`, `lastName`, and `email` are supposed to signify here? Where's the data supposed to be coming from? The BillingContact? User? Account?

Reply:
Cool, I'm pretty sure we only care about the email so I'll go ahead and remove the unused properties